sdks/python/apache_beam/coders/coder_impl.py

```diff
@@ -354,14 +359,29 @@ def decode(self, value):
 _ITERABLE_LIKE_TYPES = set()  # type: Set[Type]
 
 
+def _verify_dill_compat():
+  base_error = (
+      "This pipeline runs with the update_compatibility_version=2.67.0 flag. "
```

Comment:
nit: 

we usually call these params 'options' or 'pipeline options', not flags. You could use the same wording as below